base_layer/wallet_ffi/mobile_build.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+source build.config
+CURRENT_DIR=$TARI_REPO_PATH/base_layer/wallet_ffi
+ZMQ_REPO=https://github.com/zeromq/libzmq.git
```

Comment:
Shouldn't this go to `build.config`?

Reply:
I can move it there.

Figured this would be a fixed long term url.

Looking at it again it probably would be even better to bind to a fixed source release, what do you think?